.github/workflows/test.yml

```diff
@@ -42,23 +42,62 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        backend: [compose, k8s]
         test: [scenarios_test.py, rpc_test.py, graph_test.py, ln_test.py, dag_connection_test.py]
     steps:
       - uses: actions/checkout@v4
-      - if: matrix.backend == 'compose'
-        uses: ./.github/actions/compose
-      - if: matrix.backend == 'k8s'
-        uses: ./.github/actions/k8s
-      - if: matrix.backend == 'k8s'
-        run: |
+      - uses: hynek/setup-cached-uv@v1
+      - uses: medyagh/setup-minikube@master
+        with:
+          mount-path: ${{ github.workspace }}:/mnt/src
+      - uses: actions/download-artifact@v4
+        with:
+          name: warnet
+          path: /tmp
+      - run: |
+          echo loading the image directly into minikube docker
+          eval $(minikube -p minikube docker-env)
+          docker load --input /tmp/warnet.tar
+          docker image ls -a
+
+          echo Installing warnet python package for cli
+          uv venv
+          uv pip install -e .
+
+          echo patching k8s configs to use local image
+          echo Patching warnet-rpc-statefulset-dev to use local registry
+          sed -i "s|imagePullPolicy: Always|imagePullPolicy: Never|g" src/templates/rpc/warnet-rpc-statefulset-dev.yaml
+          sed -i "s|bitcoindevproject/warnet-rpc:dev|local/warnet-rpc:ci|g" src/templates/rpc/warnet-rpc-statefulset-dev.yaml
```

Comment:
* [`imagePullPolicy` is already `Never`](https://github.com/willcl-ark/warnet/blob/30c94200d2e42ec5ef1fe52ab4bf14e047f9e9a9/src/templates/rpc/warnet-rpc-statefulset-dev.yaml#L19)

* `bitcoindevproject/warnet-rpc:dev` isn't listed any longer

Reply:
Interesting, I guess these just, aren't required at all any more. Going to remove entirely and push to see what Ci thinks as I'm too lazy to double-check it locally somehow..

Reply:
Seems to have worked fine in https://github.com/bitcoin-dev-project/warnet/actions/runs/9530029641/job/26269437436?pr=382

Going to mark as resolved with both lines removed